core/src/test/java/org/elasticsearch/common/settings/MemorySizeSettingsTests.java

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Not sure if these tests are better here in one place or if they should be in separate test classes for each class that defines one of the settings?